custom-login/post-install.js

```diff
@@ -0,0 +1,3 @@
+const createConfigFile = require('../tools/create-config-file');
```

Comment:
Curious if you've thought about pulling this out into a shared repo. Then instead of duplicating code, we can pull it in on `postinstall`

Reply:
Perhaps, if I keep using it and I like it